src/_locales/ru/messages.json

```diff
@@ -60,76 +60,76 @@
-    "html_options_suspend_minute": { "message": "минуту" },
-    "html_options_suspend_minutes": { "message": "минуты" },
-    "html_options_suspend_hour": { "message": "час" },
+    "html_options_suspend_minute": { "message": "минуты" },
+    "html_options_suspend_minutes": { "message": "минут" },
+    "html_options_suspend_hour": { "message": "часа" },
     "html_options_suspend_hours": { "message": "часов" },
-    "html_options_suspend_day": { "message": "день" },
+    "html_options_suspend_day": { "message": "дня" },
     "html_options_suspend_days": { "message": "дней" },
-    "html_options_suspend_week": { "message": "неделю" },
-    "html_options_suspend_weeks": { "message": "недели" },
-    "html_options_suspend_no_pinned": { "message": "Никогда не приостанавливать закрепленные" },
+    "html_options_suspend_week": { "message": "недели" },
+    "html_options_suspend_weeks": { "message": "недель" },
+    "html_options_suspend_no_pinned": { "message": "Никогда не приостанавливать закрепленные вкладки" },
     "html_options_suspend_no_forms": { "message": "Никогда не приостанавливать вкладки с несохраненными полями ввода" },
     "html_options_suspend_no_audio": { "message": "Никогда не приостанавливать вкладки с проигрывающимся аудио" },
     "html_options_suspend_only_connected": { "message": "Никогда не приостанавливать вкладки в режиме оффлайн" },
     "html_options_suspend_only_on_battery": { "message": "Никогда не приостанавливать вкладки, когда подключен источник питания" },
-    "html_options_suspend_no_active_tabs": { "message": "Никогда не приостанавливать вкладки активную вкладку окна" },
-    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять вкладку при открытии" },
-    "html_options_suspend_discard_after_suspend": { "message": "Применить встроенный в Chrome механизм сохранения памяти при приостановке" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1a": { "message": "Уменьшает использование памяти приостановленной вкладки еще больше, используя" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1b": { "message": "встроенную в Chrome функциональность 'отбрасывание вкладки'. Это может привести к 500%" },
-    "html_options_suspend_discard_after_suspend_tooltip_line1c": { "message": "уменьшению использования памяти при работе с большим количеством вкладок." },
-    "html_options_suspend_discard_after_suspend_tooltip_line2a": { "message": "Однако, это может также привести к небольшой задержке отображения при выборе" },
+    "html_options_suspend_no_active_tabs": { "message": "Никогда не приостанавливать активную вкладку в каждом окне" },
+    "html_options_suspend_automatically_unsuspend": { "message": "Автоматически возобновлять приостановленную вкладку при переключении" },
+    "html_options_suspend_discard_after_suspend": { "message": "Использовать встроенный в Chrome механизм экономии памяти при приостановке вкладки" },
```

Comment:
about the work of tab-discarding. Does it start to work when I put a tab to sleep? or does it work anytime for a tab that sleeps? 
p.s. By sleep, I mean suspend

Reply:
Can't get the difference between these two questions, can u please explain? Why u asked them? What do u think may be not correct here, at which line?

Reply:
line 78. 
it can be translated as
Использовать встроенный в Chrome механизм экономии памяти при приостановке вкладки
or
Использовать встроенный в Chrome механизм экономии памяти для приостановленных вкладок

However, in practice, I see a delay in rendering only for non-suspend tabs.

Reply:
The second variant I think would be better, coz this discarding mechanism works for all tabs as in general for as I can understand now.

Reply:
Oh I see, well then let's use the first original variant:
"Использовать встроенный в Chrome механизм экономии памяти при приостановке вкладки",
so no changes here. And I see it as it's reduces memory usage of every suspended tab meaning all suspended tabs in general.

Reply:
I can't assign or use on a suspended tab discard action, only load, coz they'r already discarded, coz I have TGS option "Apply Chrome's built-in memory-saving when suspending" enabled and it works. I think discarding works for all tabs if enabled including TGS suspended ones. If discarding disabled in TGS options, suspended tabs just hang in Chrome Task manager (consuming some memory), otherwise they'r not, only active suspended tab does before I unsuspend it and it becomes the usual active tab or just loaded if lost focus. Can't get what u r talking bout, sorry, but if u have a better translation variant, let's review it, and I guess we also may need dev @deanoemcke's help in understanding how it all works here with this one phrase

Reply:
@MonsterNya This option will discard tabs AFTER they have been suspended. So it CAN discard suspended tabs. When you revisit one of the suspended discarded tabs, it will undiscard automatically, showing the suspended tab in it's place. This is where you get the rendering delay.